jetstream/certificate-transparency.toml

```
@@ -17,8 +17,8 @@ data_source = "events_certerror"
 
 [metrics.tls_timing]
 friendly_name = "TLS timing"
-description = "Amount of time for TLS handshake in seconds"
-select_expression = "COALESCE(SUM(CAST(key AS INT64)/1000000), 0)"
+description = "Amount of time for TLS handshake in miliseconds"
+select_expression = "COALESCE(SUM(CAST(key AS INT64)), 0)"
```

I think you should be safe to leave this alone, I was just testing some different things and forgot to change it back. If you do decide to leave it as-is, pretty sure it should be microseconds.

```suggestion
description = "Amount of time for TLS handshake in seconds"
select_expression = "COALESCE(SUM(CAST(key AS INT64)/1000000), 0)"
```